gui/default/syncthing/core/syncthingController.js

```diff
@@ -1532,6 +1532,12 @@ angular.module('syncthing.core')
             delete $scope.folderRejections[folder + "-" + device];
         };
 
+        $scope.ignoreRejectedFolder = function (folder, device) {
+            $scope.config.ignoredFolders.push(folder.id);
```

Comment:
event.data.folder that you are passing into this function I believe is already a string, so "string".id ends up undefined.